crates/ibc/src/core/ics02_client/events.rs

```diff
@@ -390,3 +390,90 @@ impl From<UpgradeClient> for abci::Event {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::mock::header::MockHeader;
+    use ibc_proto::google::protobuf::Any;
+    use tendermint::abci::Event as AbciEvent;
+
+    #[test]
+    fn ibc_to_abci_client_events() {
+        struct Test {
+            kind: IbcEventType,
+            event: AbciEvent,
+            expected_keys: Vec<&'static str>,
+            expected_values: Vec<&'static str>,
+        }
+
+        let client_type = ClientType::new("07-tendermint".to_string());
+        let client_id = ClientId::new(client_type.clone(), 0).unwrap();
+        let consensus_height = Height::new(0, 5).unwrap();
+        let consensus_heights = vec![Height::new(0, 5).unwrap(), Height::new(0, 7).unwrap()];
+        let header: Any = MockHeader::new(consensus_height).into();
+        let expected_keys = vec![
+            "client_id",
+            "client_type",
+            "consensus_height",
+            "consensus_heights",
+            "header",
+        ];
+        let expected_values = vec!["07-tendermint-0", "07-tendermint", "0-5", "0-5,0-7"];
```

Comment:
Oops, we're still missing the value for `header`

Reply:
oh, weird. let me fix it